packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view-container.tsx

```diff
@@ -126,15 +127,44 @@ const OnfidoSdkViewContainer = observer(
                         ],
                     });
                     setIsOnfidoInitialized(true);
+                    setIsOnfidoLoaded(true);
```

Comment:
clarification: how is_onfido_loaded different from is_onfido_initialized?

Reply:
loaded is specifically for the script to be loaded in the dom, we're using initialized as a flag which is being passed down to the components to render components